src/fsharp/FSComp.txt

```diff
@@ -1041,8 +1041,8 @@ lexHashBangMustBeFirstInFile,"#! may only appear as the first line at the start
 1186,parsInvalidProperty,"Invalid property getter or setter"
 1187,parsIndexerPropertyRequiresAtLeastOneArgument,"An indexer property must be given at least one argument"
 1188,tastInvalidAddressOfMutableAcrossAssemblyBoundary,"This operation accesses a mutable top-level value defined in another assembly in an unsupported way. The value cannot be accessed through its address. Consider copying the expression to a mutable local, e.g. 'let mutable x = ...', and if necessary assigning the value back after the completion of the operation"
-1189,parsNonAdjacentTypars,"Type parameters must be placed directly adjacent to the type name, e.g. \"type C<'T>\", not     type \"C   <'T>\""
-1190,parsNonAdjacentTyargs,"Type arguments must be placed directly adjacent to the type name, e.g. \"C<'T>\", not \"C  <'T>\""
+1189,parsNonAdjacentTypars,"Remove spaces after type, e.g. \"type C<'T>\", not type \"C   <'T>\" type parameters must be placed directly adjacent to the type name"
```

Comment:
Suggested change:

```
Remove spaces between the type name and type parameter, e.g. \"type C<'T>\", not type \"C   <'T>\". Type parameters must be placed directly adjacent to the type name
```

This way it's a bit more specific about what the issue is. It also fixes a run-on sentence.

Reply:
Did you mean to delete this line without replacing it?

-1190,parsNonAdjacentTyargs,"Type arguments must be placed directly adjacent to the type name, e.g. \"C<'T>\", not \"C  <'T>\""

Reply:
Like a twerp I looked at the conversation, not the files :-(

Reply:
@KevinRansom what does that mean? Is something wrong?

Reply:
@voronoipotato no, I nothing is wrong.